src/Components/VideoCard/VideoCard.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import { VideoDrawer } from '../VideoDrawer/VideoDrawer'
+import './videocard.css'
+import { deleteHistoryVideo } from '../../Utils/history-utils'
+import { addLikedVideo, removeLikedVideo } from '../../Utils/likevideo-utils'
+import { addToWatchlater, deleteFromWatchlater } from '../../Utils/watchlater-utils'
+import { useHistory } from '../../Context/history-provider'
+import { useAuth } from '../../Context/auth-provider'
+import { useLikedVideo } from '../../Context/likevideo-provider'
+import { useWatchlater } from '../../Context/watchlater-provider'
+import { findElementInData } from '../../Utils/common-utils'
```

Comment:
Suggestion: you can create index.js file in context and any other folder and can import it from there

Reply:
done